documentation/installation/manual_cli.md

```diff
@@ -0,0 +1,99 @@
+---
+id: manual_cli
+title: Installation, Linux command-line
+description: "Manual installation steps using the Linux command-line"
+---
+
+This guide will help you install HACS using the Linux command-line on your Home Assistant server.
+
+### Step 1 - Get the URL for the latest HACS release
+
+In your browser, visit the [HACS latest release page](https://github.com/custom-components/hacs/releases/latest).
+
+![install2](/img/install2.png)
+
+Under Assets, right-click `hacs.zip` and copy that URL to your clipboard.
+
+### Step 2 - Download the latest HACS release
+
+At your Linux command-line, type `wget ` followed by a space, then paste the URL from your clipboard. Press Enter to download `hacs.zip` to your current directory. (`xx` and `y`, below, will be the actual latest-version numbers in the URL):
+```
+user@host:~$ wget https://github.com/hacs/integration/releases/download/0.xx.y/hacs.zip
+--2020-01-01 12:58:13--  https://github.com/hacs/integration/releases/download/0.xx.y/hacs.zip
+...
+Connecting to github.com (github.com) ... connected.
+...
+HTTP request sent, awaiting response... 200 OK
+Length: ..... (nnK) [application/octet-stream]
+Saving to: ‘hacs.zip’
+
+hacs.zip 100%[==========================================>]  76.20K  --.-KB/s    in 0.09s
+
+2020-01-01 12:58:14 (... KB/s) - ‘hacs.zip’ saved [nnnnn/nnnnn]
+
+```
+
+### Step 3 - Create the destination subdirectories
+
+The `hacs` folder needs to be placed under `<config_dir>/custom_components/`
+
+`<config-dir>` is where your `configuration.yaml` file is located.
+
+On Hassio/Docker the final location will be `/config/custom_components/hacs`
+
+On Hassbian(venv) the final location will be `/home/homeassistant/.homeassistant/custom_components/hacs`
+
+For this example, `<config-dir>` will be `/path/to/homeassistant/config`
+
+Change-directory to your Home Assistant configuration directory:
+```
+user@host:~$ cd /path/to/homeassistant/config
+user@host:/path/to/homeassistant/config$ 
+```
+
+Create subdirectories, `custom_components/hacs`. If the subdirectories don't exist, this command will create them:
```

Comment:
This is not needed, you can just use the `-d` flag when unzipping.
Like `unzip hacs.zip -d hacs`